web/annotation_editor_layer_builder.css

```diff
@@ -140,6 +170,10 @@
   &:hover:not(.selectedEditor) {
     border: var(--hover-outline);
   }
+
+  &.nomouse {
+    pointer-events: none;
+  }
```

Comment:
What's the `nomouse` class, since there doesn't appear to be anything setting it and a quick web-search reveals nothing about that name either?